OmopTransformer/MeasurementMapsToValueResolver.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace OmopTransformer;
+
+internal class MeasurementMapsToValueResolver : Icd10Resolver
+{
+    public MeasurementMapsToValueResolver(IOptions<Configuration> configuration, ILogger<Icd10Resolver> logger) : base(configuration, logger)
+    {
+    }
+
+    public override string Query =>
```

Comment:
```suggestion
  public override string Query =>
          @"select
                  replace(c.concept_code, '.', '') as Code,
                  cr.concept_id_2 as concept_id
          from cdm.concept c
                  inner join cdm.concept_relationship cr
                  on c.concept_id = cr.concept_id_1
          where cr.relationship_id = 'Maps to value'
          and c.domain_id = 'Measurement'";
```

would verbatim strings be easier to format sql?